server/src/main/java/org/opensearch/rest/action/admin/indices/RestGetMappingAction.java

```diff
@@ -102,17 +102,17 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
         final GetMappingsRequest getMappingsRequest = new GetMappingsRequest();
         getMappingsRequest.indices(indices);
         getMappingsRequest.indicesOptions(IndicesOptions.fromRequest(request, getMappingsRequest.indicesOptions()));
-        TimeValue clusterManagerTimeout = request.paramAsTime("cluster_manager_timeout", getMappingsRequest.masterNodeTimeout());
+        TimeValue clusterManagerTimeout = request.paramAsTime("cluster_manager_timeout", getMappingsRequest.clusterManagerNodeTimeout());
         // TODO: Remove the if condition and statements inside after removing MASTER_ROLE.
         if (request.hasParam("master_timeout")) {
             deprecationLogger.deprecate("get_mapping_master_timeout_parameter", MASTER_TIMEOUT_DEPRECATED_MESSAGE);
             if (request.hasParam("cluster_manager_timeout")) {
                 throw new OpenSearchParseException(DUPLICATE_PARAMETER_ERROR_MESSAGE);
             }
-            clusterManagerTimeout = request.paramAsTime("master_timeout", getMappingsRequest.masterNodeTimeout());
+            clusterManagerTimeout = request.paramAsTime("master_timeout", getMappingsRequest.clusterManagerNodeTimeout());
         }
         final TimeValue timeout = clusterManagerTimeout;
-        getMappingsRequest.masterNodeTimeout(timeout);
+        getMappingsRequest.clusterManagerNodeTimeout(timeout);
```

Comment:
Glad you remembered the temporary method `parseDeprecatedMasterTimeoutParameter` in https://github.com/opensearch-project/OpenSearch/blob/237031d4d124baa1189422c6feb303dc8b3d1fc9/server/src/main/java/org/opensearch/rest/BaseRestHandler.java#L217
There are 2 classes (RestGetMappingAction and RestIndicesAction) that couldn't use the method right now, because the timeout value is required after parsing in these 2 classes, while the method `parseDeprecatedMasterTimeoutParameter` doesn't have a return value, though which is suitable for other Action classes.
Probably I'd better to modify `parseDeprecatedMasterTimeoutParameter` to let it return the timeout value to suitable for all cases including the 2 Action classes in a separate PR. 🙂